modules/apps/depot/depot-test/src/testIntegration/java/com/liferay/depot/test/util/DepotTestUtil.java

```diff
@@ -139,7 +139,7 @@ public static void withSiteMember(
 			group.getGroupId(), RoleConstants.SITE_MEMBER, unsafeConsumer);
 	}
 
-	private static boolean _isAsignableRole(String roleName) {
+	private static boolean _isAssignableRole(String roleName) {
```

Comment:
What do you think, should this typo fix be in a separate commit? :)

Reply:
yeah, I can make a different commit for this.

Reply:
done